src/select-field.jsx

```diff
@@ -51,6 +51,7 @@ const SelectField = React.createClass({
     menuItemStyle: React.PropTypes.object,
     selectedIndex: React.PropTypes.number,
     style: React.PropTypes.object,
+    labelMember: React.PropTypes.string,
```

Comment:
My bad, It was working since this component is using the DropDownMenu. That's more explicit like this. Thanks